flink-formats/flink-avro-glue-schema-registry/pom.xml

```
@@ -178,5 +192,33 @@ under the License.
 			<version>${glue.schema.registry.version}</version>
 		</dependency>
 
+		<!-- test dependencies -->
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-api-java</artifactId>
+			<version>${project.version}</version>
+			<scope>test</scope>
+			<type>test-jar</type>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-table-runtime-blink_${scala.binary.version}</artifactId>
```

This module doesn't exist since the legacy planner has been removed, and the Blink planner is now the only planner.

```suggestion
			<artifactId>flink-table-runtime_${scala.binary.version}</artifactId>
```

---

didn't realize I'd been working off a fairly old checkout and based that on some outdated code from a sibling project.  Fixed.